js/script.js

```diff
@@ -60,28 +105,45 @@ var userMetaManager = (function () {
         deleteUserMetaInformation();
     });
 
+    /**
+     * Method to add styles to the alert message dialog boxes.
+     */
+    function designAlertDialogBoxes() {
```

Comment:
please get the reference to the currently opened dialog and modify only that dialog.